k8s/ithc/common/aac/manage-case-assignment.yaml

```diff
@@ -22,6 +22,6 @@ spec:
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/aac/manage-case-assignment:prod-dc892e19
     global:
-      environment: aat
+      environment: ithc
```

Comment:
Is this file still needed

Reply:
I can't see in my local (probably git move/rename issue). Deleted now.